tooling/build/scripts/generate-sitemap.js

```diff
@@ -134,7 +134,9 @@ const processDanglingDirectory = async (fullPath, relativePath, name) => {
                   marks: [
                     {
                       type: "link",
-                      href: child.permalink,
+                      attrs: {
+                        href: child.permalink,
+                      },
```

Comment:
only relevant change because we update the `href` to be inside `attrs` now. i searched the 2 `.js` files in this directory to check if we needed to update `href`

fact-check me @dcshzj if this is enoguh